server/src/serializers/VoteSerializer.js

```diff
@@ -0,0 +1,23 @@
+class VoteSerializer {
+  static getSummary(vote) {
+    const allowedAttributes = ['voteValue']
+
+    let serializedVote = {}
+    for (const attribute of allowedAttributes) {
+      serializedVote[attribute] = vote[attribute]
+    }
+
+    return serializedVote
+  }
+
+  static getTotalSummary(votes, currentUserId) {
+    const currentUserVote = votes.find(vote => vote.userId === currentUserId)
+    const serializedUserVote = currentUserVote? VoteSerializer.getSummary(currentUserVote) : false
```

Comment:
i dont even know how this is working! this should be `currentUserVote ?` with a space!